src/client/Client.java

```diff
@@ -87,20 +96,40 @@ public Client(Game gameObj) {
         frame.add(mainPanel, BorderLayout.CENTER);
         frame.add(interactionPanel, BorderLayout.SOUTH);
 
-        interactionPanel.setBorder(BorderFactory.createLineBorder(Color.black));
+        interactionPanel.setBorder(BorderFactory.createEmptyBorder());
         if (timer != null || statsLabel != null) {
-            statusPanel.setBorder(BorderFactory.createLineBorder(Color.black));
+            statusPanel.setBorder(BorderFactory.createEmptyBorder());
         }
 
-        // Create text and next button
+        // Set the text panel
         textPanel = new JPanel();
+        textPanel.setBackground(BG_COLOR);
+
+        List<Color> colorList = Arrays.asList(new Color(53,133,255),
```

Comment:
This seems like it might belong as a class field with the rest of the color variables

Reply:
That was the original idea, but it's only being referenced twice in a single method. We can move it to be a class var for stylistic purposes if you think that looks better